scripts/kaggle/kernel-sparse-embed-smoke.py

```
@@ -0,0 +1,140 @@
+import sys
+import glob
+
+WORK = "/kaggle/working"
+REPO = "https://github.com/kodflow/3gpp-mcp"
+BRANCH = "main"
+ORT_VERSION = "1.26.0"
+
+
+def sh(c, env=None):
+    print("+", c, flush=True)
+    return subprocess.run(c, shell=True, text=True, capture_output=True, env=env or os.environ.copy())
```

Ruff (0.15.15)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)